src/jdk.jfr/share/conf/jfr/profile.jfc

```diff
@@ -605,13 +605,19 @@
     </event>
 
     <event name="jdk.ObjectAllocationInNewTLAB">
-      <setting name="enabled" control="memory-profiling-enabled-medium">true</setting>
+      <setting name="enabled" control="memory-profiling-enabled-medium">false</setting>
```

Comment:
Need to sync this with <selection>. 

Perhaps a new choice are needed "Object Allocation"

Reply:
New control attributes introduced.